ui/service.go

```diff
@@ -65,17 +68,24 @@ func New(things mainflux.ThingsServiceClient, sdk sdk.SDK) Service {
 	}
 }
 
-func (gs *uiService) Index(ctx context.Context, token string) ([]byte, error) {
-	tpl := template.New("index")
+func ParseTemplate(tmp string) (*template.Template, error) {
+	tpl := template.New(tmp)
 	tpl = tpl.Funcs(template.FuncMap{
```

Comment:
If we do not need this `New()` above, then move this `tpl.Funcs()` after `ParseFiles()` where `tpl` is created.